AirCasting/APICommunicator/FixedSession.swift

```diff
@@ -7,52 +7,78 @@
 
 import Foundation
 import Combine
+import CoreLocation
 
 class FixedSession {
-    
-    struct FixedMeasurementOutput: Codable {
-        let id: Int
-        var type: String
-        var uuid: String
-        var title: String
-        var tag_list: String
-        var start_time: String
-        var end_time: String
-        var deleted: Bool?
-        var version: Int
-        var streams: [String : StreamOutput]
+    struct FixedMeasurementOutput: Decodable, Hashable {
+        typealias ID = Int64
+        let id: ID
+        let type: SessionType
```

Comment:
Are we going to have to ensure type is of the same data type as is expected in the database (int or string)?

Reply:
SessionType has a method `init(from decoder: Decoder) throws {` that is responsible for decoding from json. So the value will decode from a string value. then an additional computed value `var rawValue: Int64` is used to map to the database (hopefully temporary solution - the final one should store string value).